src/target/tx/devo/devo7e/hardware.h

```diff
@@ -29,20 +29,24 @@
     TWO_WAY(INP_HOLD, (GPIOC, GPIO11), CHAN_INVERT) \
     TWO_WAY(INP_FMOD, (GPIOC, GPIO10), CHAN_INVERT)
 
-#define ADDON_SWITCH_CFG \
-    ADDON_SWITCH("2x2", SWITCH_2x2, 0) \
-    ADDON_SWITCH("3x1", SWITCH_3x1, 0) \
-    ADDON_SWITCH("3x2", SWITCH_3x2, 0)
-
 #define EXTRA_SWITCHES \
-    EXTRA_3WAY(INP_SWA, 0x04, 0x08, CHAN_NONINV, SWITCH_3x2) \
+    EXTRA_3WAY(INP_SWA, 0x04, 0x08, CHAN_NONINV, SWITCH_3x1) \
     EXTRA_3WAY(INP_SWB, 0x01, 0x02, CHAN_NONINV, SWITCH_3x2) \
     EXTRA_2WAY(INP_SWA, 0x04, 0x08, CHAN_INVERT, SWITCH_2x2) \
     EXTRA_2WAY(INP_SWB, 0x01, 0x02, CHAN_INVERT, SWITCH_2x2)
```

Comment:
This seems odd as well because it's no longer symmetric.  Not sure what's correct.

Reply:
the 2way switch is also broken here.  I'm not sure we actually support a 2x1 configuration actually.   Previously we did not.  I was trying to more closely align with how the 7e-256 is setup.

Reply:
Makes sense.